PassleSync.Website/App_Plugins/PassleSync/backoffice/passleSync/components/organisms/entitiesTable/entitiesTable.html

```diff
@@ -0,0 +1,36 @@
+﻿<div>
+    <umb-load-indicator ng-if="vm.loading"></umb-load-indicator>
+    <div ng-if="!vm.isLoading">
+        <div class="table-actions">
+            <button class="left-button btn-primary" ng-class="{'disabled': vm.isUpdating}" ng-click="vm.update()">Update</button>
+            <button class="left-button btn-primary" ng-class="{'disabled': vm.isUpdating || vm.unsyncedCount == 0}" ng-click="vm.sync()">Sync {{vm.selectedCount == 0 ? "all" : "selected"}}</button>
```

Comment:
For consistency with the WordPress plugin, this button should read "Sync (All|Selected) (Posts|Authors)"